fs/path/mod.ts

```diff
@@ -384,7 +381,7 @@ export const win32 = {
     }
   },
 
-  isAbsolute: function isAbsolute(path: string) {
+  isAbsolute(path: string) {
```

Comment:
Add return type. Here and elsewhere.

Reply:
Done! Usually typescript greatly infer return type but I agree better define this explicitly.